RELEASE.md

```diff
@@ -0,0 +1,122 @@
+```
+
+Please ensure that you also update the [CHANGELOG.md](./raiden-ts/CHANGELOG.md) accordingly. You should have the changes for the new version tracked as `[Unreleased]`. You only need to replace `[Unreleased]` with the new version. 
+
+### Increasing the version
+Initially you need to decide if the new release would require `patch`, `minor`, `major` upgrade. For more you can check the [npm version](https://docs.npmjs.com/cli/version) documentation.
+
+#### Using the script
+
+Then you can proceed with the version upgrade and commit creation. If you use an environment that can execute bash scripts then you can just run [prepare-release.sh](https://github.com/raiden-network/light-client/blob/master/prepare-release.sh). 
+
+```bash
+./prepare-release.sh
+```
+
+This script will bump the version according to the provided parameter and then it will create a commit with the message that looks like `0.22.0->0.22.1`.
+
+#### Manually preparing the release
+If for any reason you cannot run the script, you can proceed to prepare the release Pull Request manually.
+
+Go to the `raiden-ts` directory to start bump the package version.
+
+```bash
+cd raiden-ts
+```
+
+Then run `npm version` to bump the version for both `package.json` and `package-lock.json`. Let's assume that we prepare for a new release containing a couple of minor bug fixes. You have to run the following command: 
+
+```bash
+npm version patch
+```
+
+Assuming `raiden-ts` was at version `0.22.0`, the command will update the package files' version tag to `0.22.1`. 
+
+
+Then you need to create a new commit with the message `0.22.0->0.22.1`. After the new commit
+
+```bash
+git add package.json
+git add package-lock.json
+git commit -m '0.22.0->0.22.1'
+```
+
+## Creating a Pull Request
+After committing the CHANGELOG changes, along with the version bump you can proceed with the creation of the Pull Request.
+
+Push your local branch to GitHub and proceed to create a Pull Request.
+
+```bash
+git push origin -u prepare_release
+```
+
+You need one approval for your Pull Request to get merged to master. After merging to master, you need to tag the release.
+
+## Publishing on Github
+Since you need to deal with the GitHub release interface we strongly suggested to use for the creation of the tag too.
+
+To draft a new GitHub release, visit the [release interface](https://github.com/kelsos/light-client/releases) and press the `Draft a new release` button. Then set `v0.22.1` as the `Tag version` and keep master as the `Target`. Then use `v0.22.1` as the release title.
+
+Now you need prepare the release description. Don't forget to include the changelog entries for the specific version.
+
+Proceed to publish the new release. A new release gets automatically tagged. Tagging will trigger the CI release flow that will also publish the artifact to `npm`.
+
+## Publishing on npm
+Before we proceed to publish on npm, we need to synchronize our local master with `upstream`.
```

Comment:
Shouldn't be mentioned here that the following steps are only needed for manual publishing on `npm`, and as per above, upon GH's release, CI would do it automatically?

Reply:
While I mention it in the line above now that I look at it it can be confusing so I need to clarify it indeed

Reply:
I clarified it in the beginning now :)